input/pagecontent/authentication.md

```diff
@@ -0,0 +1,169 @@
+### Introduction
+
+Authentication verifies the identity of entities, such as healthcare professionals and organizations, within healthcare systems.
+Verified identities support downstream processes such as secure authorization (access control) and accounting (audit logging).
+
+### Problem overview
+
+Healthcare professionals often require access to patient data managed by other organizations (e.g. hospitals). Legal and regulatory frameworks restrict data sharing, requiring strong verification of:
+
+- The professional’s identity, role, and organizational affiliation.
+- The care organization’s identity.
+- The IT service provider’s identity, if involved.
+
+In small systems, these verifications can be managed through direct agreements and trusting relationships between entities (e.g., hospitals trusting each others staff and HR systems).
+However, in national-scale healthcare systems with many organizations and professionals, direct agreements become impractical.
+
+Also, traditional authentication topologies relying on a central trusted authority introduce single points of failure and may not scale well. Downtime of such a central authority can disrupt access for all dependent entities. Also, often these schemes have a use-case tailored design, providing a limited set of identity claims, making them inflexible for future use-cases. Adding new identity claims require these central authorities to expand their systems and governance, which is often a slow process.
+
+#### Requirements
+
+This IG is focused on establishing a robust authentication mechanism that scales in a national healthcare context. This national context requires a solution that is scalable, cost-effective, and user-friendly for healthcare professionals.
+Also, it must support various (not yet known) use-cases, including those without direct end-user involvement (e.g., automated systems).
+To create a robust and safe solution in a hostile global internet, the topology must avoid a single point of failure and support peer-to-peer trust, enabling direct interactions between entities without relying on the availability of third parties.
+
+To summarize, the authentication solution must:
+
+- Support portable identity claims issued by authoritative sources.
+- Support presenting a combination of claims from different authoritative sources in a single transaction.
+- Support use cases with and without direct end-user involvement.
+- Enable service providers to act on behalf of care organizations, with scope-limited authorization.
```

Comment:
Need to define "Service Provider" below. Also explain "scope-limited authorization"

Reply:
I've changed the terminology of vendor to service provider, since that is the term used.
Replaced it by "for authorized use-cases".